src/modules/fancyzones/lib/ZoneSet.cpp

```diff
@@ -560,7 +560,7 @@ bool ZoneSet::CalculateGridZones(Rect workArea, JSONHelpers::GridLayoutInfo grid
 
     // Note: The expressions below are carefully written to 
     // make the sum of all zones' sizes exactly total{Width|Height}
-    long long totalPercents = 0;
+    long totalPercents = 0;
```

Comment:
It would be better not to use `long` because on Windows `long` is actually `int` even on x64.

Reply:
I agree but long is already used everywhere in the code.

Reply:
Let's start removing it at least where we make changes/fixes.

Reply:
I suggest we leave it like this and open a separate issue to convert all longs to ints.

Reply:
Agreed, new issue to migrate

Reply:
There is no reason to introduce more cases, this was a `long long` change to `long` so this PR should not use it.